src/MusicStore/Controllers/AccountController.cs

```diff
@@ -65,6 +71,7 @@ public IActionResult Login(string returnUrl = null)
             }
             else
             {
+                _logger.LogInformation("Failed to login {userName}.", model.Email);
```

Comment:
Eh, depends on your philosophy. If you're trying to surface these to a user in some way then yes, totally a warning. But if these are for debugging and tracing then I don't think someone forgetting their password counts as a warning.